src/components/form/Wrapper.styled.js

```diff
@@ -1,65 +1,49 @@
 import styled from 'styled-components';
 
 const Wrapper = styled.div`
-	witdh: 100vw;
-	height: 100vh;
-	display: flex;
-	justify-content: center;
-
-
-
-}
-button {
-background: rgba(21, 146, 230, 0.5);
-width: 170px;
-height: 40px;
-border: 1px solid rgba(255, 0, 0, 0.8);
-margin: 0.3rem;
-
-border-radius: 2px;
-font-size: 0.9rem;
-font-family: Arial, Helvetica, sans-serif;
-letter-spacing: 0.2rem;
-color: #000000;
-cursor: pointer;
-color:white;
-
-
-}
-
-
-input {
-	border-radius: 5px;
-	margin: 0.625rem;
-	width: 220px;
-	height: 35px;
-	box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.10);
-	border: white;
-
-	font-family: Arial, Helvetica, sans-serif;
-	font-size: 0.688em;
-	padding-left: 0.5rem;
-	letter-spacing: 0.1em;
-	justify-content: center;
-
-	
-}
-section{
-	margin-left: 1.9rem;
-	margin-top: 1.9rem;
-}
-h4{
-	
-	padding-left:4rem;
-	padding-top: 2rem;
-	letter-spacing: 0.1em;
-	line-height:1.3rem;
-	padding-bottom: 2.5rem;
-	font-size: 0.83em;
-	
-	
-
-}
-
+	 {
+		width: 100vw;
+		height: 100vh;
+		display: flex;
+		justify-content: center;
+	}
+	button {
+		background: rgba(21, 146, 230, 0.5);
+		width: 9.375rem;
+		height: 2.5rem;
+		border: 1px solid rgba(255, 0, 0, 0.8);
```

Comment:
This is ok, but there is a more modern way of writing rgba in css: `rgb(21 146 230 / 0.5)`